src/Converter/Feature2Texture.js

```diff
@@ -34,94 +34,86 @@ function _drawPolygon(ctx, vertices, indices, properties = {}, style, size, exte
     }
 
     // draw line polygon
-    if (style.stroke || properties.stroke) {
-        strokeStyle(style, properties, ctx, invCtxScale);
+    if (style.stroke.color) {
+        strokeStyle(style, ctx, invCtxScale);
         ctx.stroke();
     }
 
     // fill polygon
-    if (style.fill || properties.fill) {
-        fillStyle(style, properties, ctx);
+    if (style.fill.color) {
+        fillStyle(style, ctx);
         ctx.fill();
     }
 }
 
-function fillStyle(style, properties, ctx) {
-    const fill = style.fill || properties.fill;
-    if (ctx.fillStyle !== fill) {
-        ctx.fillStyle = fill;
+function fillStyle(style, ctx) {
+    if (ctx.fillStyle !== style.fill.color) {
+        ctx.fillStyle = style.fill.color;
     }
-    const alpha = style.fillOpacity || properties['fill-opacity'] || 1.0;
-    if (alpha !== ctx.globalAlpha && typeof alpha == 'number') {
-        ctx.globalAlpha = alpha;
+    if (style.fill.opacity !== ctx.globalAlpha) {
+        ctx.globalAlpha = style.fill.opacity;
     }
 }
 
-function strokeStyle(style, properties, ctx, invCtxScale) {
-    const stroke = style.stroke || properties.stroke;
-    if (ctx.strokeStyle !== stroke) {
-        ctx.strokeStyle = stroke;
+function strokeStyle(style, ctx, invCtxScale) {
+    if (ctx.strokeStyle !== style.stroke.color) {
+        ctx.strokeStyle = style.stroke.color;
     }
-    const width = Math.round((style.strokeWidth || properties['stroke-width'] || 2.0)) * invCtxScale;
+    const width = Math.round((style.stroke.width || 2.0)) * invCtxScale;
     if (ctx.lineWidth !== width) {
         ctx.lineWidth = width;
     }
-    const alpha = style.strokeOpacity || properties['stroke-opacity'] || 1.0;
+    const alpha = style.stroke.opacity || 1.0;
     if (alpha !== ctx.globalAlpha && typeof alpha == 'number') {
         ctx.globalAlpha = alpha;
     }
 }
 
 function drawPoint(ctx, x, y, style = {}, invCtxScale) {
     ctx.beginPath();
-    const opacity = style.pointOpacity || 1.0;
+    const opacity = style.point.opacity || 1.0;
     if (opacity !== ctx.globalAlpha) {
         ctx.globalAlpha = opacity;
     }
 
-    ctx.arc(x, y, style.radius, 0, 2 * Math.PI, false);
-    ctx.fillStyle = style.fill || 'white';
-    ctx.fill();
-    ctx.lineWidth = (style.lineWidth || 1.0) * invCtxScale;
-    ctx.strokeStyle = style.stroke || 'red';
-    ctx.stroke();
+    ctx.arc(x, y, (style.point.radius || 3.0) * invCtxScale, 0, 2 * Math.PI, false);
+    if (style.point.color) {
+        ctx.fillStyle = style.point.color;
+        ctx.fill();
+    }
+    if (style.point.line) {
+        ctx.lineWidth = (style.point.width || 1.0) * invCtxScale;
+        ctx.strokeStyle = style.point.line;
```

Comment:
No more default color ?

Reply:
For the moment, no, because if `style.point.line == undefined` then the line will not drawn.